teams.md/docusaurus.config.ts

```diff
@@ -34,6 +32,9 @@ const config: Config = {
 
     markdown: {
         mermaid: true,
+        hooks: {
+            onBrokenMarkdownLinks: 'warn',
```

Comment:
any reason we wouldn't want this set to be more aggressive than a warning?

Reply:
Nope, happy to change it. That will match the `onBrokenLinks` setting too.